client/pkg/tlsutil/versions.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tlsutil
+
+import (
+	"crypto/tls"
+	"fmt"
+)
+
+// tlsVersions is a map of TLS version string to the value of tls.Config.Min/MaxVersion.
+var tlsVersions = map[string]uint16{
+	"":      0, // If version was not given use 0 (uninitialized version) to let Go decide.
+	"TLS12": tls.VersionTLS12,
+	"TLS13": tls.VersionTLS13,
```

Comment:
```suggestion
	"TLS1.2": tls.VersionTLS12,
	"TLS1.3": tls.VersionTLS13,
```

Reply:
Let's use correct semantics names, we don't want to run into Windows 9 problems.